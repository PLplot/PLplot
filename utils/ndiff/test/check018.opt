--relative-error 1.e-15

