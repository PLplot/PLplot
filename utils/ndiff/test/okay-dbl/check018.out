25c25
< 1	1.0	0	    3.81053724092472423424359245942546424600452984399700e-02
--- field 4	relative error 1.27e-15 [6*(machine epsilon)]
> 1       1.0     0           3.81053724092472e-02
26c26
< 1	1.5	0	    1.61420869583317427891530717624170848868014933501800e-02
--- field 4	relative error 2.79e-15 [13*(machine epsilon)]
> 1       1.5     0           1.61420869583317e-02
27c27
< 1	2.0	0	   -1.04593709289476912465394498315683218064600876481920e-01
--- field 4	relative error 8.62e-15 [39*(machine epsilon)]
> 1       2.0     0          -1.04593709289476e-01
28c28
< 1	2.5	0	   -4.30865150235775768384788967653055154891253883965480e-01
--- field 4	relative error 1.80e-15 [8*(machine epsilon)]
> 1       2.5     0          -4.30865150235775e-01
29c29
< 1	3.0	0	   -1.17809724509617246442349126872981358157393852476570e+00
--- field 4	relative error 2.07e-15 [9*(machine epsilon)]
> 1       3.0     0          -1.17809724509617e+00
31c31
< 1	4.0	0	   -5.83482672464782838290159808502515501790814884017110e+00
--- field 4	relative error 1.52e-15 [7*(machine epsilon)]
> 1       4.0     0          -5.83482672464782e+00
34c34
< 1	5.5	0	   -3.13399865189794493423933233329304272043089049923620e+01
--- field 4	relative error 1.59e-15 [7*(machine epsilon)]
> 1       5.5     0          -3.13399865189794e+01
35c35
< 1	6.0	0	   -3.33365661639470778781144951515885103741859211166280e+01
--- field 4	relative error 2.34e-15 [11*(machine epsilon)]
> 1       6.0     0          -3.33365661639470e+01
36c36
< 1	6.5	0	    1.57509345619882169889238418141080714128537022215480e+01
--- field 4	relative error 1.02e-15 [5*(machine epsilon)]
> 1       6.5     0           1.57509345619882e+01
38c38
< 1	7.5	0	    1.02451741212290924677698269906802760416069532590590e+03
--- field 4	relative error 9.10e-15 [41*(machine epsilon)]
> 1       7.5     0           1.02451741212290e+03
39c39
< 1	8.0	0	    3.25012722435208792081867119571056510752142251287190e+03
--- field 4	relative error 2.52e-15 [11*(machine epsilon)]
> 1       8.0     0           3.25012722435208e+03
41c41
< 1	9.0	0	    2.16047298462417274563264467697976952909938273686180e+04
--- field 4	relative error 1.35e-15 [6*(machine epsilon)]
> 1       9.0     0           2.16047298462417e+04
44c44
< 1	10.5	0	    1.31614497377234705209781285961137904387322957628200e+04
--- field 4	relative error 5.39e-15 [24*(machine epsilon)]
> 1       10.5    0           1.31614497377234e+04
45c45
< 1	11.0	0	   -4.89852834510988510707287669537856487218443638597560e+05
--- field 4	relative error 1.07e-15 [5*(machine epsilon)]
> 1       11.0    0          -4.89852834510988e+05
48c48
< 1	12.5	0	   -3.73511768108064454693583983345472997717971110290580e+07
--- field 4	relative error 1.20e-15 [5*(machine epsilon)]
> 1       12.5    0          -3.73511768108064e+07
53c53
< 1	15.0	0	    4.01189471464499590269268601351504463031905340011400e+09
--- field 4	relative error 1.43e-15 [6*(machine epsilon)]
> 1       15.0    0           4.01189471464499e+09
54c54
< 1	15.5	0	    2.81983695600612686531982990872052911128950009925900e+10
--- field 4	relative error 2.44e-15 [11*(machine epsilon)]
> 1       15.5    0           2.81983695600612e+10
55c55
< 1	16.0	0	    1.32497567691076656283159471687763331793177783506470e+11
--- field 4	relative error 4.95e-15 [22*(machine epsilon)]
> 1       16.0    0           1.32497567691076e+11
56c56
< 1	16.5	0	    5.16756332476721948646749547942796190133917178035930e+11
--- field 4	relative error 1.77e-15 [8*(machine epsilon)]
> 1       16.5    0           5.16756332476721e+11
57c57
< 1	17.0	0	    1.74770579355658371404750572085393057076063105719440e+12
--- field 4	relative error 2.10e-15 [9*(machine epsilon)]
> 1       17.0    0           1.74770579355658e+12
58c58
< 1	17.5	0	    5.04495067014915690967064247182808440084417521961550e+12
--- field 4	relative error 1.36e-15 [6*(machine epsilon)]
> 1       17.5    0           5.04495067014915e+12
59c59
< 1	18.0	0	    1.11661440152837784415317038071429605093351581530100e+13
--- field 4	relative error 7.17e-15 [32*(machine epsilon)]
> 1       18.0    0           1.11661440152837e+13
63c63
< 1	20.0	0	   -4.10950003498359839815144584196328005522767816653300e+15
--- field 4	relative error 2.07e-15 [9*(machine epsilon)]
> 1       20.0    0          -4.10950003498359e+15
64c64
< 1	20.5	0	   -1.79755844538585990185952555154632422746361485951260e+16
--- field 4	relative error 5.56e-15 [25*(machine epsilon)]
> 1       20.5    0          -1.79755844538585e+16
66c66
< 1	21.5	0	   -2.18672091001347879683873506924183766525451040870160e+17
--- field 4	relative error 3.95e-15 [18*(machine epsilon)]
> 1       21.5    0          -2.18672091001347e+17
69c69
< 1	23.0	0	    4.95706212569344523300281135807715129403304599903530e+18
--- field 4	relative error 1.03e-15 [5*(machine epsilon)]
> 1       23.0    0           4.95706212569344e+18
70c70
< 1	23.5	0	    4.67958956716544811113427104523314195189148991133680e+19
--- field 4	relative error 1.75e-15 [8*(machine epsilon)]
> 1       23.5    0           4.67958956716544e+19
71c71
< 1	24.0	0	    2.75503257408103299472543581100210883201095849185470e+20
--- field 4	relative error 1.07e-15 [5*(machine epsilon)]
> 1       24.0    0           2.75503257408103e+20
72c72
< 1	24.5	0	    1.32380435113257222531991556747217313902200222390630e+21
--- field 4	relative error 1.78e-15 [8*(machine epsilon)]
> 1       24.5    0           1.32380435113257e+21
74c74
< 1	25.5	0	    1.93056731399803288903919752085495514385129476036210e+22
--- field 4	relative error 1.52e-15 [7*(machine epsilon)]
> 1       25.5    0           1.93056731399803e+22
77c77
< 1	27.0	0	   -5.21978641835519783035196036005524031261679743698410e+23
--- field 4	relative error 1.54e-15 [7*(machine epsilon)]
> 1       27.0    0          -5.21978641835519e+23
78c78
< 1	27.5	0	   -5.50003639143347930973221784660330197483020450503310e+24
--- field 4	relative error 1.76e-15 [8*(machine epsilon)]
> 1       27.5    0          -5.50003639143347e+24
79c79
< 1	28.0	0	   -3.52525282652121410293219789381609070012546250724360e+25
--- field 4	relative error 1.22e-15 [5*(machine epsilon)]
> 1       28.0    0          -3.52525282652121e+25
80c80
< 1	28.5	0	   -1.83412549100738552469871592264520948251438015591570e+26
--- field 4	relative error 3.18e-15 [14*(machine epsilon)]
> 1       28.5    0          -1.83412549100738e+26
81c81
< 1	29.0	0	   -8.19704444982020958466047798588423546344142151159090e+26
--- field 4	relative error 1.17e-15 [5*(machine epsilon)]
> 1       29.0    0          -8.19704444982020e+26
84c84
< 1	30.5	0	   -1.21686404171193857766037295021174633668467960000280e+28
--- field 4	relative error 7.05e-15 [32*(machine epsilon)]
> 1       30.5    0          -1.21686404171193e+28
86c86
< 1	31.5	0	    1.14104141380513711157779378462708059268548623616180e+30
--- field 4	relative error 6.29e-15 [28*(machine epsilon)]
> 1       31.5    0           1.14104141380513e+30
89c89
< 1	33.0	0	    2.09676957380251636737721103509429749098413780952580e+32
--- field 4	relative error 2.92e-15 [13*(machine epsilon)]
> 1       33.0    0           2.09676957380251e+32
91c91
< 1	34.0	0	    2.71309843765917396218016909475425938708068702732990e+33
--- field 4	relative error 1.49e-15 [7*(machine epsilon)]
> 1       34.0    0           2.71309843765917e+33
93c93
< 1	35.0	0	   -3.09586923023748633142876774717116221657321537102370e+34
--- field 4	relative error 2.09e-15 [9*(machine epsilon)]
> 1       35.0    0          -3.09586923023748e+34
94c94
< 1	35.5	0	   -3.89311864881064656620633497809681533898091491158370e+35
--- field 4	relative error 1.71e-15 [8*(machine epsilon)]
> 1       35.5    0          -3.89311864881064e+35
95c95
< 1	36.0	0	   -2.86374588177905602848503417051680181182719036078640e+36
--- field 4	relative error 2.06e-15 [9*(machine epsilon)]
> 1       36.0    0          -2.86374588177905e+36
96c96
< 1	36.5	0	   -1.69640523267206620253379322761322528328486040208460e+37
--- field 4	relative error 3.76e-15 [17*(machine epsilon)]
> 1       36.5    0          -1.69640523267206e+37
98c98
< 1	37.5	0	   -3.71636207366836701629041352785420712588497786477860e+38
--- field 4	relative error 1.83e-15 [8*(machine epsilon)]
> 1       37.5    0          -3.71636207366836e+38
99c99
< 1	38.0	0	   -1.25631780622472678288834340505923393994416547753420e+39
--- field 4	relative error 5.29e-15 [24*(machine epsilon)]
> 1       38.0    0          -1.25631780622472e+39
102c102
< 1	39.5	0	    2.06748827524718600589637714984913702927009956903730e+41
--- field 4	relative error 2.81e-15 [13*(machine epsilon)]
> 1       39.5    0           2.06748827524718e+41
103c103
< 1	40.0	0	    1.61076223785882926449885269859680550549732156243370e+42
--- field 4	relative error 5.57e-15 [25*(machine epsilon)]
> 1       40.0    0           1.61076223785882e+42
104c104
< 1	40.5	0	    1.00753065255605236960906102558107388129762008794550e+43
--- field 4	relative error 2.33e-15 [11*(machine epsilon)]
> 1       40.5    0           1.00753065255605e+43
105c105
< 1	41.0	0	    5.39021041464422615797623893994742587624022980156600e+43
--- field 4	relative error 1.10e-15 [5*(machine epsilon)]
> 1       41.0    0           5.39021041464422e+43
106c106
< 1	41.5	0	    2.45489205926824572543611444214652219392951040117800e+44
--- field 4	relative error 2.42e-15 [11*(machine epsilon)]
> 1       41.5    0           2.45489205926824e+44
108c108
< 1	42.5	0	    1.51359841561331853564050654048726290364635849542580e+45
--- field 4	relative error 5.65e-15 [25*(machine epsilon)]
> 1       42.5    0           1.51359841561331e+45
109c109
< 1	43.0	0	   -1.13129703166836517504134315662791847755780610978380e+46
--- field 4	relative error 4.59e-15 [21*(machine epsilon)]
> 1       43.0    0          -1.13129703166836e+46
110c110
< 1	43.5	0	   -1.63528963315038949695468391029943291344134205916200e+47
--- field 4	relative error 5.83e-15 [26*(machine epsilon)]
> 1       43.5    0          -1.63528963315038e+47
111c111
< 1	44.0	0	   -1.34182117475088529411104563999742565546060139653150e+48
--- field 4	relative error 3.99e-15 [18*(machine epsilon)]
> 1       44.0    0          -1.34182117475088e+48
113c113
< 1	45.0	0	   -4.94990029161365904459406134012048905755207823570600e+49
--- field 4	relative error 1.68e-15 [8*(machine epsilon)]
> 1       45.0    0          -4.94990029161365e+49
114c114
< 1	45.5	0	   -2.36548850141861510429691686843938075432410995211140e+50
--- field 4	relative error 2.11e-15 [9*(machine epsilon)]
> 1       45.5    0          -2.36548850141861e+50
116c116
< 1	46.5	0	   -1.64053242579010973556373073199232819805578130974510e+51
--- field 4	relative error 5.87e-15 [26*(machine epsilon)]
> 1       46.5    0          -1.64053242579010e+51
117c117
< 1	47.0	0	    1.21070277032116772667749503279163207549758852062960e+52
--- field 4	relative error 6.59e-15 [30*(machine epsilon)]
> 1       47.0    0           1.21070277032116e+52
118c118
< 1	47.5	0	    1.85814676112634250638705451158768315817364915822510e+53
--- field 4	relative error 1.26e-15 [6*(machine epsilon)]
> 1       47.5    0           1.85814676112634e+53
119c119
< 1	48.0	0	    1.59839261899346293311548366383637481826094810242270e+54
--- field 4	relative error 1.92e-15 [9*(machine epsilon)]
> 1       48.0    0           1.59839261899346e+54
120c120
< 1	48.5	0	    1.09849655379763215986631621472238985519979002692930e+55
--- field 4	relative error 1.98e-15 [9*(machine epsilon)]
> 1       48.5    0           1.09849655379763e+55
122c122
< 1	49.5	0	    3.21866926979323754913089450427943937048928004509080e+56
--- field 4	relative error 2.30e-15 [10*(machine epsilon)]
> 1       49.5    0           3.21866926979323e+56
123c123
< 1	50.0	0	    1.26445092328997415688789240813430768098074999549750e+57
--- field 4	relative error 3.31e-15 [15*(machine epsilon)]
> 1       50.0    0           1.26445092328997e+57
124c124
< 1	50.5	0	    2.48188014579934655102285461857210546700951075957620e+57
--- field 4	relative error 2.67e-15 [12*(machine epsilon)]
> 1       50.5    0           2.48188014579934e+57
125c125
< 1	51.0	0	   -1.81532773381955888938023605216777313400108422783200e+58
--- field 4	relative error 4.91e-15 [22*(machine epsilon)]
> 1       51.0    0          -1.81532773381955e+58
127c127
< 1	52.0	0	   -2.64397241909489337063051914598108746665637304501430e+60
--- field 4	relative error 1.21e-15 [5*(machine epsilon)]
> 1       52.0    0          -2.64397241909489e+60
128c128
< 1	52.5	0	   -1.89373099307065273917252663743616747769456720882680e+61
--- field 4	relative error 1.36e-15 [6*(machine epsilon)]
> 1       52.5    0          -1.89373099307065e+61
129c129
< 1	53.0	0	   -1.15716272274907744883182951512406131215103585713240e+62
--- field 4	relative error 6.51e-15 [29*(machine epsilon)]
> 1       53.0    0          -1.15716272274907e+62
131c131
< 1	54.0	0	   -2.46485369600276985145189393898385471619589842360630e+63
--- field 4	relative error 4.00e-15 [18*(machine epsilon)]
> 1       54.0    0          -2.46485369600276e+63
133c133
< 1	55.0	0	    3.71475960154697794210545844063247919730043868793930e+64
--- field 4	relative error 2.20e-15 [10*(machine epsilon)]
> 1       55.0    0           3.71475960154697e+64
134c134
< 1	55.5	0	    6.33556711657763715671499543052686238022162867460510e+65
--- field 4	relative error 1.18e-15 [5*(machine epsilon)]
> 1       55.5    0           6.33556711657763e+65
136c136
< 1	56.5	0	    4.40837600649441670138788702269825572758282165297350e+67
--- field 4	relative error 1.63e-15 [7*(machine epsilon)]
> 1       56.5    0           4.40837600649441e+67
137c137
< 1	57.0	0	    2.79697241530011546969576768187367667025923526906140e+68
--- field 4	relative error 2.05e-15 [9*(machine epsilon)]
> 1       57.0    0           2.79697241530011e+68
139c139
< 1	58.0	0	    6.42962430540484809471620796937642988205415637761170e+69
--- field 4	relative error 1.31e-15 [6*(machine epsilon)]
> 1       58.0    0           6.42962430540484e+69
140c140
< 1	58.5	0	    1.40112532730468681699404828153113066947355181979910e+70
--- field 4	relative error 4.81e-15 [22*(machine epsilon)]
> 1       58.5    0           1.40112532730468e+70
141c141
< 1	59.0	0	   -1.01441020704820192972721332695077863629033567497230e+71
--- field 4	relative error 1.93e-15 [9*(machine epsilon)]
> 1       59.0    0          -1.01441020704820e+71
142c142
< 1	59.5	0	   -1.81324365959010890453915440957231133503275305649010e+72
--- field 4	relative error 4.76e-15 [21*(machine epsilon)]
> 1       59.5    0          -1.81324365959010e+72
143c143
< 1	60.0	0	   -1.75971495472908891589771643038651673376219882791710e+73
--- field 4	relative error 4.99e-15 [22*(machine epsilon)]
> 1       60.0    0          -1.75971495472908e+73
147c147
< 1	62.0	0	   -2.20050094213494735551532131281587169171863994027390e+76
--- field 4	relative error 3.36e-15 [15*(machine epsilon)]
> 1       62.0    0          -2.20050094213494e+76
154c154
< 1	65.5	0	    2.12772209860943964749043560818466275968431751139020e+82
--- field 4	relative error 4.43e-15 [20*(machine epsilon)]
> 1       65.5    0           2.12772209860943e+82
156c156
< 1	66.5	0	    2.31142465284221268969674680514568874875099866305820e+83
--- field 4	relative error 1.17e-15 [5*(machine epsilon)]
> 1       66.5    0           2.31142465284221e+83
157c157
< 1	67.0	0	   -1.66732023541175400005979056738511458853114521429580e+84
--- field 4	relative error 2.46e-15 [11*(machine epsilon)]
> 1       67.0    0          -1.66732023541175e+84
159c159
< 1	68.0	0	   -3.36761018263802777939498745433022294919592487445120e+86
--- field 4	relative error 2.30e-15 [10*(machine epsilon)]
> 1       68.0    0          -3.36761018263802e+86
160c160
< 1	68.5	0	   -2.77010449656429660939853685192077158143478685840170e+87
--- field 4	relative error 2.39e-15 [11*(machine epsilon)]
> 1       68.5    0          -2.77010449656429e+87
161c161
< 1	69.0	0	   -1.93971555218801719155844766376162837301956122922660e+88
--- field 4	relative error 3.64e-15 [16*(machine epsilon)]
> 1       69.0    0          -1.93971555218801e+88
162c162
< 1	69.5	0	   -1.15573905219609190080408075584519429776527374169560e+89
--- field 4	relative error 1.59e-15 [7*(machine epsilon)]
> 1       69.5    0          -1.15573905219609e+89
164c164
< 1	70.5	0	   -1.34783696863714343135850192081091166833218144876440e+90
--- field 4	relative error 2.68e-15 [12*(machine epsilon)]
> 1       70.5    0          -1.34783696863714e+90
166c166
< 1	71.5	0	    1.96546272893996737763038265276685614619802160795000e+92
--- field 4	relative error 3.83e-15 [17*(machine epsilon)]
> 1       71.5    0           1.96546272893996e+92
167c167
< 1	72.0	0	    2.10402339491044870814886498159667991665780458416440e+93
--- field 4	relative error 3.96e-15 [18*(machine epsilon)]
> 1       72.0    0           2.10402339491044e+93
168c168
< 1	72.5	0	    1.78248063320388180119971197980968099176470400492530e+94
--- field 4	relative error 1.04e-15 [5*(machine epsilon)]
> 1       72.5    0           1.78248063320388e+94
169c169
< 1	73.0	0	    1.28494666194880422027151269709363796594923265811110e+95
--- field 4	relative error 3.35e-15 [15*(machine epsilon)]
> 1       73.0    0           1.28494666194880e+95
174c174
< 1	75.5	0	   -1.48643181031121153294460848822314377787206636672950e+99
--- field 4	relative error 1.14e-15 [5*(machine epsilon)]
> 1       75.5    0          -1.48643181031121e+99
### Maximum relative error in matching lines = 9.68e-16 at line 151 field 4
